Gemfile

```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 gemspec
 
 %w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
-  branch = ENV.fetch('BRANCH','3-1-maintenance')
+  branch = ENV.fetch('BRANCH','3-9-maintenance')
```

Comment:
Can you add this to the travis matrix instead?

Reply:
I am sorry, but this is out of my scope.

Reply:
Its literally a case of adding
```
env:
  - BRANCH='3-1-maintenance'
  - BRANCH='3-9-maintenance'
```
To the file, we need to support older rspec and newer rspec.

Reply:
That is just the first step. The next step would be to make the test compatible between RSpec 3.9+ and older. My only intentions were:

1. Fix the Fedora packaged rspec-its, because we have only RSpec 3.9 in Fedora.
2. Point out that the dependency could be updated.

Reply:
This gem primarily exists to support users transitioning out of 2.x into 3.x where this functionality was removed.  Any updated tests have to be in addition to the support for the original version.

Reply:
I understand that this is for updates. OTOH, if they are updating, they probably won't update to 3.1 but they'll go straight to the 3.9.